jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/PathMappings.java

```diff
@@ -173,19 +273,9 @@ public MatchedResource<E> getMatched(String path)
                     {
                         if (_optimizedExact)
                         {
-                            int i = path.length();
-                            while (i >= 0)
-                            {
-                                MappedResource<E> candidate = _exactMap.getBest(path, 0, i--);
-                                if (candidate == null)
-                                    continue;
-
-                                matchedPath = candidate.getPathSpec().matched(path);
-                                if (matchedPath != null)
-                                {
-                                    return new MatchedResource<>(candidate.getResource(), candidate.getPathSpec(), matchedPath);
-                                }
-                            }
+                            MappedResource<E> exact = _exactMap.get(path);
```

Comment:
There are no test cases that enter this condition.

Reply:
After the commit with the `testOptimalExactIterative` this line is still not being hit